components/resources/ai_chat_ui_strings.grdp

```diff
@@ -48,4 +48,31 @@
   <message name="IDS_CHAT_UI_RETRY_BUTTON_LABEL" desc="A button label to retry API again">
      Retry
   </message>
+  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_DEFAULT" desc="AI Chat intro message for the default model">
+    I'm here to help. What can I assist you with today?
+  </message>
+  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_LEO_EXPANDED" desc="AI Chat intro message for the expanded model">
+    I have a vast base of knowledge and a large memory able to help with more complex challenges.
+  </message>
+  <message name="IDS_CHAT_UI_INTRO_MESSAGE_CHAT_LEO_CLAUDE_INSTANT" desc="AI Chat intro message for the Claude Instant model">
+    Hi! My name is Claude. I was created by Anthropic to be helpful, harmless, and honest.
+  </message>
```

Comment:
Are these used?

Reply:
yes, they are used in the f/e

Reply:
```cpp
      {"introMessage-chat-default", IDS_CHAT_UI_INTRO_MESSAGE_CHAT_DEFAULT},
      {"introMessage-chat-leo-expanded",
       IDS_CHAT_UI_INTRO_MESSAGE_CHAT_LEO_EXPANDED},
      {"introMessage-chat-claude-instant",
       IDS_CHAT_UI_INTRO_MESSAGE_CHAT_LEO_CLAUDE_INSTANT},
```

Reply:
I saw them being mapped, but I didn't see the mapped strings (like `introMessage-chat-default`) being used elsewhere.